website/public/locales/en/error.json

```diff
@@ -0,0 +1,16 @@
+{
+  "default": "Something went wrong!",
+  "err_1000": "Ivalid task type",
+  "err_1001": "Failed to acknowledge task",
+  "err_1002": "Failed to not acknowledge task",
+  "err_1003": "Invalid response type",
+  "err_1004": "Interaction request failed",
+  "err_1005": "Failed to generate task",
+  "err_1006": "No tasks of type {{task_type}} are currently available.",
+  "err_1007": "Task availability query failed",
+  "err_1008": "Message size too long",
+  "err_1009": "User recent messages have duplicates",
+  "err_1010": "Message text is empty",
+  "err_1011": "User cannot reply twice to the same message",
+  "err_1012": "User has too many pending tasks"
```

Comment:
I just copied these messages from the backend. Please suggest more friendly messages

Reply:
@andreaskoepf can you help on this?

Reply:
we could add what the user could do, for example "Current  has too many pending tasks, please wait for a couple of minutes and try again."

Reply:
I would show the error message that the backend sends in default case, e.g. I think this is better than "Something went wrong!". Regarding more explanatory error messages: Sure, could be done in a separate PR if someone is interested in analyzing when the error are generated and writing a more explanatory text, but IMO the strings as they are now are fine for v1.